Source/swiftlint/Commands/VersionCommand.swift

```diff
@@ -9,9 +9,9 @@
 import Commandant
 import Foundation
 import Result
+import SwiftLintFramework
 
-private let version = Bundle(identifier: "io.realm.SwiftLintFramework")!
-    .object(forInfoDictionaryKey: "CFBundleShortVersionString")!
+private let version = Version.current.value
```

Comment:
We can just use `Version.current.value` directly in `VersionCommand.run(_:)`, so you can remove this constant.